cli/internal/specs/v0/destination.go

```diff
@@ -18,6 +28,8 @@ type Destination struct {
 	// Destination plugin PK mode
 	PKMode PKMode `json:"pk_mode,omitempty" jsonschema:"default=default"`
 
+	ExternalSyncGroupId string `json:"external_sync_group_id,omitempty"`
```

Comment:
I don't really mind, but why have `external` in the name, and not call it `sync_group_id`?

Reply:
I was trying to be specific with the fact it's an "external" value, I'll rename it